python/ray/train/lightning/lightning_trainer.py

```diff
@@ -178,10 +178,8 @@ class LightningTrainer(TorchTrainer):
     using the arguments provided in ``trainer_init_config`` and then run
     ``pytorch_lightning.Trainer.fit``.
 
-    TODO(yunxuanx): make this example testable
-
     Example:
-        .. code-block:: python
+        .. testcode::
```

Comment:
how does this work? can you explain this a bit for me? I am curious.

Reply:
I am also new to this. I asked Justin and Matt before, and it seems that the logic is hidden quite deeply. As far as I know, the code block within this tag is extracted and tested using an `m5.2xlarge` instance. The easiest way to obtain the testing configuration is to go to the Buildkite page and check the Environment tab.